torchtitan/config/job_config.py

```diff
@@ -408,6 +408,13 @@ class Checkpoint:
     enable: bool = False
     """Whether to enable checkpoint"""
 
+    enable_ft: bool = True
+    """
+    Checkpoints data loader state if enabled. Otherwise infers the data loader
```

Comment:
The description is a little bit vague to me, how does this option helps checkpoint?Do I have to enable FT.enabled to use this?